cclib/method/ddec.py

```diff
@@ -711,6 +956,20 @@ def _phiai(self, ya, bigphiAI, atomi):
 
         return phiAI, cond_density
 
+    def _phiaii(self, rhowavg, eta, bigphiAI, atomi):
```

Comment:
Code comment for what this does, referring back to the paper?

Reply:
I've added a comment that refers back to the paper. Thanks!